nav2_dwb_controller/dwb_controller/src/dwb_controller.cpp

```diff
@@ -149,6 +151,16 @@ void DwbController::followPath()
 
     rclcpp::Rate loop_rate(100ms);
     while (rclcpp::ok()) {
+      if (action_server_ == nullptr) {
```

Comment:
Not sure we need to check if the server is active at each iteration, but we do need to active it on the node transition.